src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    @include font(12px, 500, 18px);
+
+    color: $text-main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @include font(10px, 400, 14px);
+
+    color: $text-secondary-color;
+    margin: 0;
+  }
+
+  &__reviews {
+    @include flex(center, space-between);
+
+    margin-bottom: 24px;
+    margin-top: 16px;
+  }
+
+  &__reviw {
+    @include font(10px, 400, 14px);
+
+    color: $text-main-color;
+  }
+
+  &__price {
+    @include flex('', space-between);
```

Comment:
The first parameter in the 'flex' mixin is an empty string. If the mixin expects a specific value for alignment, you should provide it. Otherwise, consider removing the empty string.